src/dodal/devices/fast_grid_scan.py

```diff
@@ -251,7 +252,7 @@ class FastGridScan(Device):
     expected_images = Component(Signal)
 
     # Kickoff timeout in seconds
-    KICKOFF_TIMEOUT: float = 5.0
+    KICKOFF_TIMEOUT: float = 60.0
```

Comment:
Should: This was just increased to debug https://github.com/DiamondLightSource/hyperion/issues/1101, right? I think we can revert to 5s